test/e2e/storage/sanity.go

```diff
@@ -294,6 +294,22 @@ var _ = Describe("sanity", func() {
 			}, "3m", "5s").Should(Equal(capacity.AvailableCapacity), "total capacity after controller restart")
 		})
 
+		It("should return right capacity", func() {
+			resp, err := cc.GetCapacity(context.Background(), &csi.GetCapacityRequest{
+				AccessibleTopology: &csi.Topology{
+					Segments: map[string]string{
+						"pmem-csi.intel.com/node": nodeID,
+					},
+				},
+			})
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
This is an anti-pattern - always include an additional explanation.
https://github.com/kubernetes/kubernetes/issues/34059

To a lesser extend that also applies to other assertions.